packages/worker/src/worker.ts

```diff
@@ -919,7 +925,11 @@ export class Worker<T extends WorkerSpec = DefaultWorkerSpec> {
       'console.log',
       (...args: any[]) => {
         if (workflow.info.isReplaying) return;
-        console.log(`${workflow.info.workflowType} ${workflow.info.runId} >`, ...args);
+        if (this.options.workflowConsoleLog !== undefined) {
```

Comment:
We should pass workflow info into the console log function, otherwise it's a bit useless for anything other than dropping logs.
Info also contains `isReplaying` which will let users decide if they want to see logs from replays or not.

Reply:
Also, I'm pretty sure that for production code users will use a proper logger instead of using console.log.

Reply:
Should I just undo this then? I'm not using the noisy workflow any longer anyway

Reply:
Let's just undo for now